src/generated/resources/data/forge/tags/worldgen/biome/is_hot.json

```diff
@@ -0,0 +1,20 @@
+{
+  "replace": false,
+  "values": [
+    "minecraft:desert",
+    "minecraft:nether_wastes",
```

Comment:
A friend and I were talking and they pointed out that it is odd that nether biomes are in a hot tag. Specifically as this makes it harder for mods with json structures to use as they only take a single biome tag. So if they want a structure to spawn in hot biomes but only in overworld, they cannot use this tag. Ideally, tags could do a union system and only import entries that exists in all tags in a given list (like overworld and hot tag together). 

Since union doesn't exist, maybe the nether biomes should be split off of here to a hot_otherwordly biome tag?

Reply:
Could just do nested tags like forge does with ingots. `forge:hot/overworld`, `forge:hot/nether`, etc.